dags.yaml

```diff
@@ -240,3 +240,12 @@ bqetl_org_mozilla_fenix_derived:
     retry_delay: 30m
     start_date: '2020-10-18'
   schedule_interval: daily
+
+bqetl_google_analytics_derived:
+  schedule_interval: 0 16 * * *
+  default_args:
+    owner: bewu@mozilla.com
+    email: ['bewu@mozilla.com']
```

Comment:
George, do you still want to get notifications for these queries?

Reply:
Nope. Looking to make my escape.